### ISSUE TITLE
Publicar el código de descarga y procesamiento de subtítulos

### ISSUE THREAD

**Issue by dcabo** [civio/verba#95]
El script que se descarga los subtítulos cada noche está en un repo separado, `verba-rtve`, que ahora mismo es bastante caótico: junta Python con Ruby, y el código para descargar los subtítulos está demasiado enganchado con el que los convierte a frases y con el que luego los mete en Elastic. No es bonito. 🤷‍♂️ Pero es necesario abrirlo para trabajar en los issues de NLP aquí.

@esebastian estoy pensando en copiar el repo `verba-rtve` a este, y meterlo bajo `rtve`, que es como lo tenemos desplegado en `midas`. Inicialmente la idea era tener una "herramienta genérica" por un lado y un "caso de ejemplo" por otro (y había motivos para ello), pero en la práctica es todo RTVE y no creo que esté partido con una línea clara.

**Comment by esebastian** [comment]
@dcabo Habrá que hacer algunos cambios en el workflow que actualiza los subtítulos y en la configuración necesaria para que siga ejecutándose en `midas`. Si quieres le puedo pegar yo un ojo.

**Comment by dcabo** [comment]
Creo que antes tengo que hacer un poco de limpieza en `civio-rtve` para quitar las cosas de prueba y tal. Te aviso cuando esté listo para migrar.

**Comment by rubiojr** [comment]
Parece que el backend de Verba (verba.civio.es) ha dejado de funcionar, me he dado cuenta por que estaba (ab)usando de él para sincronizar los subtítulos desde https://github.com/rubiojr/verba-go, así que me lié a escribir un scrapper el finde, para evitar la dependencia del servicio.

Aprovecho para enlazarlo: https://github.com/rubiojr/rtve-go

**Comment by dcabo** [comment]
Sí, perdona, hemos tenido un problema administrativo con Elastic, que nos cede el entorno, y no he tenido tiempo de moverlo a nuestro servidor mientras lo arreglamos (que espero que sea pronto). Espero que sea cuestión de días, perdona!

**Comment by rubiojr** [comment]
No hay ningún problema David, faltaría mas!

**Comment by dcabo** [comment]
@rubiojr ya debería funcionar todo de nuevo. Está en nuestro entorno, que igual va más lento, pero es algo temporal mientras arreglamos el entorno principal. Perdona las molestias!
